memcached_stat.py

```diff
@@ -8,29 +9,24 @@
 # Global settings, one entry per instance
 CONFIGS = []
 
+class EmptyString:
```

Comment:
It seems kinda weird to me that bmemcached is considering that an empty string is equivalent to None (and bad !)